src/Categories/CategoryEntity.php

```diff
@@ -0,0 +1,35 @@
+<?php
+namespace FurnitureStoreApi\Categories;
+use FurnitureStoreApi\Services\HeaderService;
+HeaderService::setHeaders();
+
+class CategoryEntity implements \JsonSerializable
+{
+    private int $id;
+    private string $name;
+    private int $products;
```

Comment:
I know that this is called 'products' in the resulting JSON, but it would be more descriptive to call it 'productCount' in this entity if you can, just to give people a better idea of what it is.